### PR TITLE
responsiveness of carousel

### PR DESCRIPTION
**Pull request by Ankita297**
## What is the change?
responsiveness of carousel
## Related issue?
close: #864 

## How was it tested?
I have tested it in my local system 
## Checklist:
Before you create this PR, confirm all the requirements listed below by checking the checkboxes `[x]`:

- [x] Have you followed the [Contribution Guidelines](https://github.com/ALPHAVIO/BlogSite/blob/master/CONTRIBUTING.md) while contributing.
- [x] Have you checked there aren't other open [Pull Requests](https://github.com/ALPHAVIO/BlogSite/pulls) for the same update/change?
- [x] Did you lint your code before making the Pull Request?
- [x] Have you made corresponding changes to the documentation?
- [x] Your submission doesn't break any existing feature.
- [x] Have you tested the code before submission?

## Screenshots or Video:
Add a screenshot or demo video if appropriate.
WordNook - Google Chrome 13-05-2021 21_03_20
WordNook - Google Chrome 13-05-2021 21_09_52

### CODE REVIEW

**Comment by Ankita297** [comment]
MINGW64__f_BlogSite 13-05-2021 21_17_26
Sir i have run npm run lint but still ci checks are failing

File: public/css/styles.css [hunk 1]
```diff
@@ -932,10 +932,12 @@ input[name='perPage'] {
 
 .testimonials-clean .people {
 	padding: 3.125rem 0 1.25rem;
+	display: flex;
+	justify-content: center;
 }
 
 .testimonials-clean .item {
-	margin-bottom: 32px;
+	margin-bottom: 34px;
```
Comment (Pranav016) [inline public/css/styles.css:940]: can you add a bootstrap class like `mb-3` (value accordingly) to the items for margin bottom or make this value in rem.

**Review by Pranav016 — CHANGES_REQUESTED** [review]

File: public/css/styles.css (hunk 1 above)
Comment (Ankita297) [reply public/css/styles.css:940]: I will make them in rem

**Comment by Ankita297** [comment]
done the changes

**Review by Pranav016 — APPROVED** [review]

**Pull request merged**